ansible/configs/kni-osp/files/ocp-install.sh.j2

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+export OSP_PROJECT="{{ osp_project_name }}"
+export GUID="{{ guid }}"
+export ORIG_PWD=$(pwd)
+export VERSION={{ my_ocp_version }}
+export SUBVER=`echo "${VERSION:0:3}"`
+export PATH=$PATH:$(pwd)
+export CMD=openshift-baremetal-install
+export EXTRACT_DIR=$(pwd)
+export PULLSECRET=/home/cloud-user/pull-secret.json
+export OFFLINE={{ my_ocp_offline }}
+
+############ DO NOT TOUCH BEYOND THIS POINT ##############
+setup_services() {
+  sudo cp `pwd`/dhcp/dhcpd.conf /etc/dhcp/dhcpd.conf
+  sudo systemctl enable dhcpd --now
+  sudo cp `pwd`/named/named.conf /etc/named.conf
+  sudo chmod 640 /etc/named.conf
+  sudo chown root:named /etc/named.conf
+  sudo restorecon -RFv /etc/named.conf
+  sudo cp `pwd`/named/0.20.10.in-addr.arpa /var/named
+  sudo cp `pwd`/named/{{ my_ocp_domain }}.zone /var/named
+  sudo systemctl enable named --now
+  openstack --os-cloud=$OSP_PROJECT subnet unset --dns-nameserver 8.8.8.8 $GUID-appnet-subnet
```

Comment:
did you consider running $GUID through jinja filter here and replacing since you are already templating? Rather than relying on the env var when it runs.

Reply:
Trying to keep my scripts as generic as I can there ... env var being set at the beginning of the script based on what jinja brings during install it never is an issue when script actually runs